.github/workflows/dapr-pubsub.yaml

```diff
@@ -1,18 +1,18 @@
-name: dapr-pubsub
```

Comment:
since you are renaming everything, does this yaml need to be renamed too?

Reply:
good call, ty!

Reply:
Updated the file name.